conda-recipe/build.sh

```diff
@@ -11,18 +11,37 @@ export ICXCFG="$(pwd)/icpx_for_conda.cfg"
 if [ -e "_skbuild" ]; then
     ${PYTHON} setup.py clean --all
 fi
-export CMAKE_GENERATOR="Ninja"
-SKBUILD_ARGS="-- -DCMAKE_C_COMPILER:PATH=icx -DCMAKE_CXX_COMPILER:PATH=icpx -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON"
-echo "${PYTHON} setup.py install ${SKBUILD_ARGS}"
-
-if [ -n "${WHEELS_OUTPUT_FOLDER}" ]; then
-    # Install packages and assemble wheel package from built bits
-    WHEELS_BUILD_ARGS="-p manylinux2014_x86_64 --build-number ${GIT_DESCRIBE_NUMBER}"
-    ${PYTHON} setup.py install bdist_wheel ${WHEELS_BUILD_ARGS} ${SKBUILD_ARGS}
-    cp dist/dpctl*.whl ${WHEELS_OUTPUT_FOLDER}
-else
-    # Perform regular install
-    ${PYTHON} setup.py install ${SKBUILD_ARGS}
+
+export CC=icx
+export CXX=icpx
+
+export CMAKE_GENERATOR=Ninja
+# Make CMake verbose
+export VERBOSE=1
+
+CMAKE_ARGS="${CMAKE_ARGS} -DDPCTL_LEVEL_ZERO_INCLUDE_DIR=${PREFIX}/include/level_zero"
+
+# -wnx flags mean: --wheel --no-isolation --skip-dependency-check
+${PYTHON} -m build -w -n -x
```

Comment:
Can we use `python -m wheel unpack`, remove copies of `libDPCTLSyclInterface.so*` and pack it back?

Reply:
Then numba_dpex won't link to those libraries for wheel install.